app/syncinfo.go

```diff
@@ -0,0 +1,84 @@
+package app
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	replicaClient "github.com/openebs/jiva/replica/client"
+	"github.com/openebs/jiva/types"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+func SyncInfoCmd() cli.Command {
+	return cli.Command{
+		Name:      "syncinfo",
+		ShortName: "ls",
+		Action: func(c *cli.Context) {
+			if err := getSyncInfo(c); err != nil {
+				logrus.Fatalf("Error running ls command: %v", err)
+			}
+		},
+	}
+}
+
+func getSyncInfo(c *cli.Context) error {
+	controllerClient := getCli(c)
+
+	reps, err := controllerClient.ListReplicas()
+	if err != nil {
+		return err
+	}
+
+	var info *types.SyncInfo
+	format := "%v\t%v\t%v\t%v\n"
+	tw := tabwriter.NewWriter(os.Stdout, 0, 20, 1, ' ', 0)
```

Comment:
why to use tabwriter? can't fmt printf helps?

Reply:
tabwriter package is useful in pretty printing the output. It takes care of padding and all. in fmt.printf we need to adjust for the small names and their sizes.